File: movieApp/app/src/main/java/com/lbc/practice/movieapp/view/favorite/FavoriteListContract.java
package com.lbc.practice.movieapp.view.favorite;

import android.content.Context;

import com.lbc.practice.movieapp.adapter.PlayingListContract;
import com.lbc.practice.movieapp.model.PlayingMovieResult;

public interface FavoriteListContract {

    interface FavoriteView {
        void moveToSecond(PlayingMovieResult.ResultsBean resultsBean);

    }

    interface FavoritePre {
        void setPlayingAdaterModel(PlayingListContract.PlayingModel model);
        void setPlayingAdaterView(PlayingListContract.View view);
        void favoriteInfo(Context context);
    }
}

File: movieApp/app/src/main/java/com/lbc/practice/movieapp/view/favorite/FavoriteListContract.kt
package com.lbc.practice.movieapp.view.favorite;

import android.content.Context;

import com.lbc.practice.movieapp.adapter.PlayingListContract;
import com.lbc.practice.movieapp.data.PlayingMovieResult;


interface FavoriteListContract {

    interface FavoriteView {
        fun moveToSecond(resultsBean: PlayingMovieResult.ResultsBean)

    }

    interface FavoritePre {
        fun setPlayingAdaterModel(model: PlayingListContract.PlayingModel)
        fun setPlayingAdaterView(view: PlayingListContract.View)
        fun favoriteInfo(context: Context)
        fun takeView(view: FavoriteListContract.FavoriteView)
    }
}